huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_cassandra_instance.go

```diff
@@ -607,7 +624,16 @@ func resourceGeminiDBInstanceV3Delete(d *schema.ResourceData, meta interface{})
 	return nil
 }
 
-func resourceGeminiDBInstanceV3Update(d *schema.ResourceData, meta interface{}) error {
+func resourceGaussDBCassandraInstanceUpdate(d *schema.ResourceData, meta interface{}) error {
+	defaults := defaultValues{
+		Mode:      "Cluster",
+		dbType:    "cassandra",
+		dbVersion: "3.11",
```

Comment:
`defaults.logName` is empty in Update function

Reply:
fixed